test/e2e/live_cluster/test_all.py

```diff
@@ -97,8 +97,8 @@ class TableRenderTests(unittest.TestCase):
         ("show statistics xdr dc"),
         ("show statistics xdr namespace"),
         ("show latencies -v"),
-        # ("show distribution time_to_live"), # TODO: Causing issues on github actions
-        # ("show distribution object_size"),
+        ("show distribution time_to_live"),  # TODO: Causing issues on github actions
+        ("show distribution object_size"),
```

Comment:
Do these still cause issues in Github actions?

Reply:
I am hoping it is fixed. Only time will tell. The failures were intermittent before

Reply:
ok